packages/angular_devkit/build_webpack/src/utils.ts

```diff
@@ -13,16 +13,24 @@ export interface EmittedFiles {
   file: string;
   initial: boolean;
   extension: string;
+
+  // if several builds are created at once for differential serving
+  // this points to the index of the configuration that was used
+  configIndex?: number;
 }
 
-export function getEmittedFiles(compilation: webpack.compilation.Compilation): EmittedFiles[] {
+export function getEmittedFiles(
```

Comment:
We need to know later, which configuration -- the es5 or the es2015 config -- was used to produce this file. Ofc, we could also determine this somewhere else, but what I want to prevent is having two data structures like ``EmittedFiles`` and ``EmittedFilesWithConfigIndex``.

What do you think?

Reply:
I don't think this should the responsibility of the webpack runner, it should be handled by the browser builder.

You are using `combineLatest` with always respects the order, hence you know what emits what.